cf/read_write/um/umread.py

```diff
@@ -3567,6 +3590,7 @@ def file_open(self, filename):
             byte_ordering=g.get("byte_ordering"),
             word_size=g.get("word_size"),
             fmt=g.get("fmt"),
+            parse=parse,
```

Comment:
I think it would be useful if, in this method or otherwise (but just above this call seems like a good place to do it), we add a `logging` call to highlight whether `parse` is enabled or disabled. Maybe even at warning level to warn when we are not parsing the contents. If you agree, please can you add in a brief line?

Reply:
Sorry (as per discussion today) this is based upon a misunderstanding of mine - the parsing is not a user configuration choice, but purely under-the-hood. So ignore me here, marking as resolved.